_src/_assets/js/dashboard-charts.js

```diff
@@ -427,7 +440,10 @@
         .append('div')
         .classed('chart', true)
         .classed('no-y-axis-domain', true)
+      const stayAtHomeOrder = stayAtHomeOrders[state.key]
+      const annotations = stayAtHomeOrder ? [{ date: parseDate(stayAtHomeOrder) }] : null
       const chart = d3AreaChart({
+        annotations,
```

Comment:
ahh looks like you are passing `null` for annotations here, but not everywhere else this function is called.